### DISCUSSION TITLE
fields from multiple tables that are independent and display them in a new one column

### DISCUSSION THREAD

**Discussion by Khalilsqu** [pocketbase/pocketbase#3943] [Q&A] [answered]
`SELECT id, email AS emails FROM student
UNION ALL
SELECT id, email AS emails FROM advisor`

Why do I get an error with this query when building a view collection?

**Comment by ganigeorgiev** [comment, marked as answer]
Root level unions are not supported at the moment (see https://github.com/pocketbase/pocketbase/discussions/1938, https://github.com/pocketbase/pocketbase/discussions/3873).

As a workaround you can try to wrap it in a subquery:

```sql
-- the cast is needed because otherwise the expression will be treated as `json` field
SELECT id, CAST(email as TEXT) as emails
FROM (
    SELECT id, email FROM student
    UNION ALL
    SELECT id, email FROM advisor
)
```

**Reply by Khalilsqu** [reply]
thank you, it worked
